src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentSum = 40 * days;
```

Comment:
in this scope you can add variables for 50 and 20
` const bigDiscount = 50;`
`const smallDiscount = 20;`